pkg/controllers/provisioning/scheduling/metrics.go

```diff
@@ -57,6 +57,19 @@ var (
 			schedulingIDLabel,
 		},
 	)
+	UnfinishedWorkSeconds = opmetrics.NewPrometheusGauge(
+		crmetrics.Registry,
+		prometheus.GaugeOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: schedulerSubsystem,
+			Name:      "unfinished_work_seconds",
```

Comment:
should this just be runtime_seconds?
```suggestion
			Name:      "runtime_seconds",
```

Reply:
Discussed offline. Trying to map this to upstream's `workqueue_unfinished_work_seconds`.